cmake/Platform/Windows/Packaging/Tests/conftest.py

```diff
@@ -50,8 +50,10 @@ def __init__(self, request, temp_dir):
 
         self.cmake_runtime_path = self.install_root / 'cmake/runtime'
         self.engine_bin_path = self.install_root / 'bin/Windows/profile/Default'
-        self.project_build_path = self.project_path / 'build/Windows'
-        self.project_bin_path = self.project_build_path / 'bin/profile'
+        self.project_build_path_profile = self.project_path / 'build/Windows'
+        self.project_build_path_release = self.project_path / 'build/Windows_release'
```

Comment:
Should probably name this folder Windows_mono instead of `Windows_release` since, "release" is a configuration of the monolithic build artifacts that already gets its own subfolder.